micromobility_toolset/model.py

```diff
@@ -492,8 +491,9 @@ def skims(self):
                 modified = True
 
                 self.logger.info(f"skimming {cost_attr} skim from network...")
-                matrix = self.network.get_skim_matrix(
-                    node_ids=self.zone_nodes, cost_attr=cost_attr, max_cost=max_cost
+                matrix, truncated = self.network.get_skim_matrix(
```

Comment:
_truncated_ is a bool matrix based on distance skims (above distance threshold), which is used to top-code generalized cost skims